onnxruntime/test/python/test_pytorch_export_contrib_ops.py

```diff
@@ -123,6 +129,13 @@ def test_gelu(self):
         x = torch.randn(3, 3)
         self.run_test(model, x, custom_opsets={"com.microsoft": 1})
 
+    @unittest.skipIf(_torch_version_lower_than("1.12"), "Gelu's approximate parameter unsupported in PyTorch < 1.12")
+    @parameterized.parameterized.expand([("default_approximate", "none"), ("tanh_approximate", "tanh")])
+    def test_gelu_supports_approximate_param(self, _, approximate: str):
+        model = torch.nn.GELU(approximate=approximate)  # type: ignore
```

Comment:
curious: why does this line bother mypy?

Reply:
torch.nn.GELU's param was introduced in the current unreleased version of pytorch. So I think it is because the pytorch version in my environment (for ort) is not new enough.

Reply:
please add a comment explaining that.

Reply:
Done